src/ucs/config/parser.c

```diff
@@ -1420,6 +1421,51 @@ static ucs_status_t ucs_config_parser_get_sub_prefix(const char *env_prefix,
     return UCS_OK;
 }
 
+void ucs_config_parse_config_files()
+{
+    char config_files[5][MAXPATHLEN] = {{0}};
+    char *dlpath_dup = NULL, *ucx_config_dir = NULL;
+    int i;
+    Dl_info ucs_dl_info;
+
+    /* Global config dir */
+    strcpy(config_files[0], UCX_GLOBAL_CONFIG_FILE);
+
+    /* Library dir */
+    if (dladdr(ucs_config_parse_config_files, &ucs_dl_info) != 0) {
```

Comment:
maybe reuse `ucs_debug_get_lib_path` [rename it to ucs_get_lib_path and move it from debug.c to sys.c]